pyproject.toml

```diff
@@ -21,19 +21,18 @@ classifiers=[
     "Operating System :: MacOS",
     "Operating System :: POSIX :: Linux",
     "Programming Language :: Python :: 3 :: Only",
-    "Programming Language :: Python :: 3.8",
```

Comment:
we are removing python 3.8 here, but not in `requires-python` below, is that on purpose ?